Scoop/NetworkManager.swift

```diff
@@ -21,4 +21,32 @@ class NetworkManager {
             ]
             return headers
         }
+    
+    static func searchLocation(depatureDate: String, startLocation: String, endLocation: String, completion: @escaping (RideResponse) -> Void) {
```

Comment:
This should not be `static`. With the singleton model we create a one static instance of the `NetworkManager` but still use instance functions. You shouldn't ever have to make a function static in NetworkManager

Reply:
Got it. I remembered seeing some of Pear's network functions marked as `static`. Is there a specific reason why in its case?